src/core/auth.ts

```diff
@@ -169,13 +169,13 @@ export default class Auth {
     })
   }
 
-  setUserToken (token) {
+  setUserToken (token, refreshToken?) {
```

Comment:
We may better keep core abstract  about scheme/token implementation  but looks good for now 😊